<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetWindowTitleClass</class>
 <widget class="QMainWindow" name="SetWindowTitleClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SetWindowTitle</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="RunButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>RunButton</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="OriginEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="TargetEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="OriginlistView">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>471</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>571</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="FindButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>FindButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HoldButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>HoldName</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="setwindowtitle.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>RunButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>SetWindowTitleClass</receiver>
   <slot>OnRunButtonClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnRunButtonClicked(bool)</slot>
 </slots>
</ui>
